WolvenKit/Views/Templates/RedVector4Editor.xaml

```diff
@@ -22,40 +22,50 @@
             <ColumnDefinition Width="20" />
             <ColumnDefinition />
         </Grid.ColumnDefinitions>
-        
+
+        <!-- X -->
         <TextBlock
             Grid.Column="0"
             HorizontalAlignment="Center"
             VerticalAlignment="Center"
             Text="X" />
-        <syncfusion:DoubleTextBox 
+        <syncfusion:DoubleTextBox
+            KeyboardNavigation.TabIndex="1"
             Grid.Column="1"
-            NumberDecimalDigits="4"
-            Value="{Binding XValue, Mode=TwoWay, UpdateSourceTrigger=PropertyChanged}" />
```

Comment:
...

Reply:
That must have snuck in, rest is good to go